From fdf40fa6828af6fffb5976f43c8f331eac8eafe7 Mon Sep 17 00:00:00 2001
From: scx <scx.mail@gmail.com>
Date: Fri, 15 Feb 2019 21:25:52 +0100
Subject: [PATCH] CMake: Remove --thread option for Vala Compiler

The --thread option is now deprecated.
Should be ignored, but actually it leads to a build failure.
---
 CMakeLists.txt | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 69d4aaa..eecfb32 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -86,8 +86,10 @@ CUSTOM_VAPIS
 OPTIONS
   #  --enable-experimental
     --target-glib=2.32 # Remember to keep this updated.
-    --thread
-   # -X -rdynamic
+  #  The --thread option is now deprecated.
+  #  Should be ignored, but actually it leads to a build failure.
+  #  --thread
+  #  -X -rdynamic
   #  -g
 )
 
